data/phi0474/phi012/__cts__.xml

```diff
@@ -1,17 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ti:work xml:lang="lat" xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:latinLit:phi0474" urn="urn:cts:latinLit:phi0474.phi012" projid="latinLit:phi012">
-    <ti:title xml:lang="eng">For Rabirius on a Charge of Treason</ti:title>
+    <ti:title xml:lang="lat">Pro C. Rabirio Perduellionis Reo Ad Quirites</ti:title>
     
            <ti:edition urn="urn:cts:latinLit:phi0474.phi012.perseus-lat2" workUrn="urn:cts:latinLit:phi0474.phi012" xml:lang="lat">
                <ti:label xml:lang="lat">Pro Rabirio Perduellionis Reo Ad Quirites</ti:label>
-               <ti:description xml:lang="mul">Cicero. M. Tulli Ciceronis. Orationes, Vol. IV. Clark, Albert Curtis, editor. Oxford: Clarendon Press, 1909.</ti:description>
+               <ti:description xml:lang="mul">Cicero. M. Tulli Ciceronis. Orationes, Volume 4. Clark, Albert Curtis, editor. Oxford: Clarendon Press, 1909.</ti:description>               
```

Comment:
I use Roman numerals when the books do. Are we changing from that now? Also, use "Vol." since it makes for a concise display in the reader.

Reply:
hi @lcerrato I thought we had always changed it before, or at least that is what I did for all of CSEL and for the first 10 Latin volumes of Cicero and in other cts_.xml  Latin and Greek files I'm sure. I had also spelled out volume in the other Cicero cts_.xml files as well (approved in a long ago pull request last year, sorry I lost track of Cicero). Using the Arabic volume number vs. Roman is what is stated here (https://github.com/PerseusDL/tei-conversion-tools/wiki/cts-textgroup-and-work-metadata-files) but I also remember you noting the example above had a Roman numeral.

Reply:
We use the Arabic within the text header, but I stick with the print in the metadata. 
That guide is wrong since it is contradicted by the Plutarch example itself I used there as well as the catalog and World Cat, I think?
I just find it easier to keep the metadata close to the print because then I'm not having to refer to contradictory guides that I myself messed up!

Reply:
hey @lcerrato you know if it's ok at this point, I would just sort of like to finish off Cicero, make sure all of the bumped Latin files have standardized cts and header metadata, and then I can do a final pull request and change it all to Roman numerals and Vol. instead of volume at the end if that the preferred practice now. I figure we can always revisit Vol.volume with a global change at some point.